src/fem_domain/domain/mesh/p_refinement.rs

```diff
@@ -259,6 +259,49 @@ impl fmt::Display for PRef {
     }
 }
 
+/// A pair of expansion orders for an `Elem`
+///
+/// Used to directly set an `Elem`'s expansion order rather than using a `PRef` to modify it
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub struct Orders {
+    i: u8,
+    j: u8,
```

Comment:
If you're working with the present logic, then [NonZeroU8](https://doc.rust-lang.org/std/num/struct.NonZeroU8.html) is possible. But it's not clear to me why zeroth order (piecewise constants) is never allowed at this level.

Reply:
With the basis functions/boundary conditions used in the library, the constant terms are handled by combinations of the linear terms, rather than including them directly:
```
\             /
 \     +     /     =   _______ 
  \         /                  

```

However I'd like to leave the door open for other types of basis functions by sticking with `u8` instead of `NonZeroU8`